updated pubs and people (#48)
muon-spectroscopy-computational-project/web: pubs.md

* [Triple Magnetic Stacking in an Iron-Containing Cuprate with Cu–Fe–Cu Magnetic Blocks](https://pubs.acs.org/doi/10.1021/acs.chemmater.4c00588)\
**_Chem. Mater. 36, 17, 8199–8207, (2024)_**\
In this work, we performed a combined experimental-theoretical study of the iron-substituted FeSr2YCu2O7.08 cuprate, which displays long-range magnetic ordering below TN ∼ 140 K, involving both Fe3+ cations in the charge reservoir block (CRB) and Cu2+ cations in the CuO2 planes. The magnetic structure comprises antiferromagnetic in-plane coupling within the FeO and the CuO2 layers, but a ferromagnetic coupling along the stacking direction.  The observation of this new magnetic structure for a M-1212 type cuprate opened up new possibilities for exploring the interplay between magnetic and superconducting interactions in classical layered cuprates

* [Hidden magnetism uncovered in a charge ordered bilayer kagome material ScV<sub>6</sub>Sn<sub>6</sub>](https://www.nature.com/articles/s41467-023-43503-9)\
**_Nat Commun 14, 7796, (2023)_**\

> Here, we present the discovery of an unnexpected hidden magnetism of the charge orger for the recently discovered charge ordered kagome material - ScV<sub>6</sub>Sn<sub>6</sub> using advanced muon-spin rotation spectroscopy. An enhancement was observed in the internal field width sensed by the muon ensemble, which takes place within the charge ordered state. Furthermore, the muon spin relaxation rate below the charge ordering temperature is substantially enhanced by applying an external magnetic field. Taken together with the hidden magnetism found in AV<sub>3</sub>Sb<sub>5</sub> (A = K, Rb, Cs) and FeGe kagome systems, our results suggest ubiqitous time-reversal symmetry-breaking in charge ordered kagome lattices.

Reviewer: ```suggestion
Here, we present the discovery of an unexpected hidden magnetism of the charge order for the recently discovered charge ordered kagome material - ScV<sub>6</sub>Sn<sub>6</sub> using advanced muon-spin rotation spectroscopy. An enhancement was observed in the internal field width sensed by the muon ensemble, which takes place within the charge ordered state. Furthermore, the muon spin relaxation rate below the charge ordering temperature is substantially enhanced by applying an external magnetic field. Taken together with the hidden magnetism found in AV<sub>3</sub>Sb<sub>5</sub> (A = K, Rb, Cs) and FeGe kagome systems, our results suggest ubiquitous time-reversal symmetry-breaking in charge ordered kagome lattices.
```

* [Low-temperature magnetic crossover in the topological kagome magnet TbMn<sub>6</sub>Sn<sub>6</sub>](https://www.nature.com/articles/s42005-022-00885-4)\
**_Communications Physics volume 5, Article number: 107, (2022)_**\
Here, we explore magnetic correlations in the recently identified topological kagome system TbMn<sub>6</sub>Sn<sub>6</sub> using muon spin rotation, combined with local field analysis and neutron diffraction. Our studies identify an out-of-plane ferrimagnetic structure with slow magnetic fluctuations which exhibit a critical slowing down below T<sup>*</sup><sub>C1</sub> ≃ 120 K and finally freeze into static patches with ideal out-of-plane order below T<sub>C1</sub> ≃ 20 K.

* [Observation of a molecular muonium polaron and its application to probing magnetic and electronic states](https://journals.aps.org/prb/abstract/10.1103/PhysRevB.104.064429)\
**_Phys.Rev.B 104,064429, (2021)_**\
Here, we give evidence for the detection of a muonium state that propagates in a molecular semiconductor lattice via thermally activated dynamics: a muonium polaron. 

> * [Mudirac: A Dirac equation solver for elemental analysis with muonic X‐rays](https://onlinelibrary.wiley.com/doi/full/10.1002/xrs.3212)\

Reviewer: ```suggestion
* [MuDirac: A Dirac equation solver for elemental analysis with muonic X‐rays](https://onlinelibrary.wiley.com/doi/full/10.1002/xrs.3212)\
```

**_X-Ray Spectrometry,1–17, (2020)_**\

> In this work, we present mudirac: a new open‐source software for the integration of the radial Dirac equation developed specifically for muonic atoms. The software can be used to predict frequencies and transitions probabilities between levels of the muonic atom, which are useful for the interpretation of muonic X‐ray spectra in elemental analysis. 

Reviewer: ```suggestion
In this work, we present MuDirac: a new open‐source software for the integration of the radial Dirac equation developed specifically for muonic atoms. The software can be used to predict frequencies and transitions probabilities between levels of the muonic atom, which are useful for the interpretation of muonic X‐ray spectra in elemental analysis. 
```

**_J. Phys. Chem. C, 124, 18, 9656–9664, (2020)_**\
Phenylacetylene-capped silicon nanoparticles have attracted interest as a novel thermoelectric material. In this paper, we report a combined muon spectroscopic (μSR) and computational study of this material in solution to investigate the microscopic electronic structure of this system.

* [Comparison between Density Functional Theory and Density Functional Tight Binding approaches for finding the muon stopping site in organic molecular crystals](https://aip.scitation.org/doi/10.1063/1.5085197)\
**_The Journal of Chemical Physics, 150, 154301, (2019)_**\
Expanding on our previous use of AIRSS to find the muon stopping site, this paper focuses on its application to organic molecules and shows how the tight binding software DFTB+, combined with the `3ob-3-1` parameter set, can produce satisfying results for it in a fraction of the time required by DFT simulations.

* [Computational Prediction of Muon Stopping Sites Using AIRSS](https://aip.scitation.org/doi/pdf/10.1063/1.5024450 )\
**_The Journal of Chemical Physics, 148, 134114, (2018)_**\

> The stopping site of the muon in a muon-spin relaxation experiment (µ+SR) is generally unknown and there few techniques that can be used to determine the muon stopping site. In this work, we propose a purely theoretical method that can be used to predict muon stopping sites in crystalline materials using only computer simulations.

Reviewer: ```suggestion
The stopping site of the muon in a muon-spin relaxation experiment (µ+SR) is generally unknown and there are few techniques that can be used to determine the muon stopping site. In this work, we propose a purely theoretical method that can be used to predict muon stopping sites in crystalline materials using only computer simulations.
```
